laplace/baselaplace.py

```diff
@@ -694,6 +694,62 @@ def _glm_forward_call(
                 "Prediction path invalid. Check the likelihood, pred_type, link_approx combination!"
             )
 
+    def sample(
```

Comment:
Random thought and I probably don't remember enough context, but it seems like it would make sense for `BaseLaplace` to inherit from `abc.ABC` and use `@abc.abstractmethod` for cases like this one.